resolwe/flow/executors/manager_comm.py

```diff
@@ -0,0 +1,66 @@
+"""Utility functions for communicating with the manager."""
```

Comment:
This file has a strange name `manager_comm`? Can you use a full name instead?

Reply:
Fixed.